Frontend/src/app/views/dashboard/profile/profile.component.scss

```diff
@@ -1,5 +1,7 @@
+@import "../../../../app/styles/display.scss";
+
 .container {
-  padding: 0px 20px 20px 20px;
+  padding: 20px;
 }
```

Comment:
Can probably just remove this and use the padding that the shared `.container` gives

Reply:
padding removed